lib/pure/times.nim

```diff
@@ -416,18 +417,32 @@ when not defined(JS):
 
 when not defined(JS):
   # C wrapper:
+  when defined(freebsd) or defined(netbsd):
```

Comment:
For some reason OpenBSD has this lines in <time.h>
```
#if __XPG_VISIBLE >= 300
extern int daylight;
extern long timezone;
#endif
```
So its not affected to this issue...
But if you want we can add one more define because OpenBSD is also support `tm_gmtoff` member.